scripts/BUILD.bazel

```diff
@@ -70,3 +77,23 @@ kt_jvm_binary(
         "//scripts/src/java/org/oppia/android/scripts/regex:regex_pattern_validation_check_lib",
     ],
 )
+
+kt_jvm_binary(
+    name = "proto_check",
+    testonly = True,
+    main_class = "org.oppia.android.scripts.regex.ProtoClassKt",
+    runtime_deps = ["//scripts/src/java/org/oppia/android/scripts/regex:proto_class_lib"],
+)
+
+kt_jvm_library(
```

Comment:
This & below: these should also be marked as testonly.

Reply:
Done.
Out of curiosity, what libraries and binaries should be marked as Testonly? 
I initially thought that only the ones which are needed for testing purposes should be marked as Testonly.